src/Table/Table.php

```diff
@@ -90,6 +90,14 @@ public function getPerPage(Request $request): ?int
         return $request->input($this->getKey().'.per_page');
     }
 
+    /**
+     * Get the per page options.
+     */
+    public function getPerPageOptions(): array
+    {
+        return array_values(array_unique(array_filter(array_merge([$this->query?->getModel()?->getPerPage()], [15, 25, 50, 100]))));
```

Comment:
@iamgergo This can only be read backward.
`array_values(array_unique(array_filter(array_merge(` 👈🏻
Would you merge a PR with Laravel's `Arr`?

Reply:
Well, no. The answer would be: Please use collections.